Add missing crate readmes [Rebase & FF] (#983)
OpenDevicePartnership/patina: components/patina_adv_logger/README.md

# Patina Advanced Logger

> Patina Advanced Logger supplies the Patina core with a logging capability to memory and serial.

Reviewer: ```suggestion
The Patina Advanced Logger provides the Patina core with in-memory and serial logging capabilities.
```
